src/main/java/com/nike/cerberus/auth/connector/okta/statehandlers/MfaStateHandler.java

```diff
@@ -0,0 +1,117 @@
+package com.nike.cerberus.auth.connector.okta.statehandlers;
```

Comment:
This class has a bunch of the POC test code from the command line script that isn't or shouldn't be being used.

Please remove the code that isn't being used by CMS.

Such as the System.out.readLine() and code related to Push, Call and SMS.

Reply:
- [ ] remove unused test code.

Reply:
Kept only used code in MfaStateHandler.java